_data/people.yml

```
@@ -142,6 +142,20 @@
       icon: linkedin
       name: Linkedin
 
+- name: Oleksandr Yardas
+  pic: /img/people/yardaso.jpeg
+  dates: 2020-present
+  role: Graduate Research Assistant
+  description: 'Olek is a Masters student interested in passive safety and operations, microreactors, and decentralized power generation. He is still trying to figure out how to integrate these interests into a viable career.'
```

I had the impression you were aiming for a PhD, ultimately. If so, perhaps "graduate" rather than "Masters"

```suggestion
  description: 'Olek is a graduate student interested in passive safety and operations, microreactors, and decentralized power generation. He is still trying to figure out how to integrate these interests into a viable career.'
```

---

good catch! I pushed this change to my branch.